src/almanack/metrics/data.py

```diff
@@ -1183,3 +1194,66 @@ def find_doi_citation_data(repo: pygit2.Repository) -> Dict[str, Any]:
                 LOGGER.warning(f"Error during OpenAlex exact DOI lookup: {e}")
 
     return result
+
+
+def compute_almanack_score(
+    almanack_table: List[Dict[str, Union[int, float, bool]]]
+) -> float:
+    """
+    Computes an Almanack score by normalizing boolean Almanack
+    table metrics in order to summarize analysis.
+
+    Args:
+        almanack_table (List[Dict[str, Union[int, float, bool]]]):
+            A list of dictionaries containing metrics.
+            Each dictionary must have a "result" key
+            with a value that is an int, float, or bool.
+            A "sustainability_correlation" key is
+            included for values to specify the
+            relationship to sustainability:
+            - 1 (positive correlation)
+            - 0 (no correlation)
+            - -1 (negative correlation)
+
+    Returns:
+        float:
+            The computed almanack score, normalized between 0 and 1.
```

Comment:
Is this still true?

Reply:
Thanks, no, needs an update -will apply in a further commit!

Reply:
Added a commit here to fix.